.pre-commit-config.yaml

```diff
@@ -3,6 +3,7 @@ repos:
   rev: v4.3.0
   hooks:
   - id: check-added-large-files
+    args: ['--maxkb=1500']
```

Comment:
I changed this to make the pre-commit pass. I am not sure what a good default could be

Reply:
I am guessing this is for the notebook. I am not sure, we could exclude all notebooks or the docs folder from the check, but I think this is probably better, it will force us to avoid very big notebooks. We can start with this and modify it as necessary.